R/dw.R

```diff
@@ -16,9 +18,8 @@
 
 
 
-dw <- function(param,gwl) {
-    data(drempelwaarden)
-    x <- subset(drempelwaarden,select=param,gwbident==gwl)[[param]]
+dw <- function(param,gwl,data=drempelwaarden) {
```

Comment:
De dataset 'drempelwaarden' moet dan wel geladen zijn in de environment. Als dat niet het geval is krijg je hier een error. Dit zou je even moeten checken in een 'kale' environment, waarin je niet `data(drempelwaarden)` hebt aangeroepen.

Reply:
Gecontroleerd. De dataset 'drempelwaarden' is beschikbaar zodra het KRWTrends package ingeladen is (zonder `data(drempelwaarden)` te gebruiken).